apps/passport-server/test/pcdpass.spec.ts

```diff
@@ -40,11 +38,6 @@ describe("pcd-pass functionality", function () {
     expect(status).to.eq(true);
   });
 
-  step("should not have a pretix service running", async function () {
-    const status = await waitForPretixSyncStatus(application);
-    expect(status).to.eq(PretixSyncStatus.NoPretix);
-  });
-
```

Comment:
I can't see any harm from disabling this test, but see other comment about whether it's even necessary to have the Pretix service in PCDPass.